### DISCUSSION TITLE
Where do Converted Files go?

### DISCUSSION THREAD

**Discussion by MikeBins** [openaudible/openaudible#909] [General]
Hello, I am a supporter of this app and I have a question that may be simple and I am not seeing it.

When I convert a file where does the conversion go or does it remain as a converted file with the old .aax marker at the end of the file name?

Thank you

**Comment by openaudible** [comment]
The converted file goes in your library, which is usually under "books". The file is converted from AAX to either MP3 or M4B (depends on preferences.) 

Select the converted book and go to the Action Menu and select "Show MP3" or "Show M4B" to see where it is.
The AAX file is not changed.

**Reply by MikeBins** [reply]
well shoot...  thank you...  my apologies for the obvious.

I love the program.   for me I would like to choose what directory or folder the converted files go into.  It would make it easier for me but knowing where they are now I can move them to where I want them.

Thank you

**Reply by openaudible** [reply]
You can change where they go in preferences. Although once you change the location, you’ll need to move the books directory over to the new location.
